R/bundle.R

```diff
@@ -198,9 +197,8 @@ createAppManifest <- function(appDir,
   }
 
   # indicate whether this is a quarto app/doc
-  if (!is.null(appMetadata$quartoInfo) && !isCloudServer) {
-    manifest$quarto <- appMetadata$quartoInfo
-  }
+  manifest$quarto <- appMetadata$quartoInfo
```

Comment:
When `appMetadata$quartoInfo` is null, we do not write a Quarto object, correct? The null assignment is interpreted as a no-op?

Reply:
Right:

``` r
x <- list()
x$x <- NULL
x
#> list()
```

<sup>Created on 2023-06-28 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

Reply:
Thanks. I didn't look to see how `manifest` was constructed.